test/extensions/tracers/opentelemetry/samplers/cel/cel_sampler_test.cc

```diff
@@ -0,0 +1,185 @@
+#include <string>
+
+#include "envoy/extensions/tracers/opentelemetry/samplers/v3/cel_sampler.pb.h"
+
+#include "source/extensions/tracers/opentelemetry/samplers/cel/cel_sampler.h"
+#include "source/extensions/tracers/opentelemetry/span_context.h"
+
+#include "test/mocks/server/tracer_factory_context.h"
+
+#include "gmock/gmock.h"
+#include "gtest/gtest.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace Tracers {
+namespace OpenTelemetry {
+
+class CELSamplerTest : public testing::Test {
+public:
+  CELSamplerTest() {
+    envoy::config::core::v3::TypedExtensionConfig typed_config;
+    const std::string yaml = R"EOF(
+    name: envoy.tracers.opentelemetry.samplers.cel
+    typed_config:
+      "@type": type.googleapis.com/envoy.extensions.tracers.opentelemetry.samplers.v3.CELSamplerConfig
+      expression:
+        parsed_expr:
```

Comment:
Please use `cel_expr_parsed` to avoid test errors in compile time options as it enforces new use of deprecated fields.

Reply:
can you show me a example of converting from [CEL canonical](https://github.com/google/cel-spec/tree/master/proto/cel/expr) to [CEL v1alpha1](https://github.com/googleapis/googleapis/tree/master/google/api/expr/v1alpha1)?